src/mongo/MongoTripodConfig.class.php

```diff
@@ -296,6 +318,14 @@ protected function loadConfig(Array $config)
                 {
                     throw new MongoTripodConfigException("View spec does not contain " . _ID_KEY);
                 }
+                if(!isset($spec['from']) || !in_array($spec['from'], $this->getPods($storeName)))
```

Comment:
You've got a ```validateTableSpec``` function to validate table specs, but validation of view specs (and search specs) are done inline. Worth moving to a function to keep it consistent?